data/mods/Magiclysm/recipes/recipe_potions.json

```diff
@@ -36,7 +36,7 @@
     "result": "mana_potion_lesser",
     "qualities": [ { "id": "CONTAIN", "level": 1 }, { "id": "MANA_INFUSE", "level": 1 } ],
     "tools": [ [ [ "surface_heat", 10, "LIST" ] ], [ [ "rune_alchemist", -1 ] ] ],
-    "components": [ [ [ "lotus", 1 ] ], [ [ "potion_starter", 1 ] ] ],
```

Comment:
all of these are magic potions so i'd appreciate if they could stay the same.

Reply:
I see, I will add the BYPASS_MASS_CHECK flag to the magic potions.